.circleci/config.yml

```diff
@@ -71,6 +71,30 @@ jobs:
     steps:
       - common_test_steps
 
+  GraphQL Types:
+    description: "Assert generated GraphQL types are up to date"
+    executor: { name: oss/node, tag: '14' }
+    steps:
+      - oss/install_specific_npm_version
+      - checkout
+      # Install Rust
+      - rust/install:
+          version: stable
+
+      # Setup the Cargo env in CircleCI's BASH_ENV file.
+      #
+      # It's unfortunate that the rust/install Orb doesn't do this since this
+      # is a CircleCI-ism.  I've opened a PR in hopes of avoiding this,
+      # though it's possible that it's out of scope (or I'm doing smth wrong):
+      #
+      #   => https://github.com/CircleCI-Public/rust-orb/pull/3
+      #   => https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command
+      #
```

Comment:
Fwiw, my PR that removes this requirement will land soon, hopefully! https://github.com/CircleCI-Public/rust-orb/pull/3

Reply:
I love when we upstream our workarounds!